kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/UploadSourceTask.kt

```diff
@@ -15,6 +15,7 @@ import org.radarbase.connect.upload.util.VersionUtil
 import org.slf4j.LoggerFactory
 
 class UploadSourceTask : SourceTask() {
+    private var pollRecordSize = 1
```

Comment:
`pollRecordSize` is not used yet

Reply:
My bad. Didn't mean to add it to the commit